docs/latest/extensions/modules/sop-class-handler.md

```diff
@@ -66,9 +66,9 @@ getSopClassHandlerModule({ servicesManager, commandsManager }) {
         displaySetInstanceUid: 0,
         wadoRoot: study.getData().wadoRoot,
         wadoUri: instance.getData().wadouri,
-        sopInstanceUid: instance.getSOPInstanceUID(),
-        seriesInstanceUid: series.getSeriesInstanceUID(),
-        studyInstanceUid: study.getStudyInstanceUID(),
+        SOPInstanceUID: instance.getSOPInstanceUID(),
+        SeriesInstanceUID: series.getSeriesInstanceUID(),
+        StudyInstanceUID: study.getStudyInstanceUID(),
```

Comment:
I understand that we want consistency here; it is a bit odd to have different casing across props for the SOP Class Handler definition.